queries/atm.py

```diff
@@ -0,0 +1,571 @@
+                voice_street_name,
+            )
+        elif len(opening_hours) is not 0 and opening_hours.startswith("opnunartím"):
+            answ_fmt = "{0}{1} fylgir {2}."
+            answer = answ_fmt.format(
+                ans_start,
+                street_name,
+                NounPhrase(opening_hours).dative,
+            )
+            voice = answ_fmt.format(
+                ans_start,
+                voice_street_name,
+                NounPhrase(opening_hours).dative,
+            )
+        elif len(opening_hours) is not 0 and opening_hours.startswith("opið"):
+            answ_fmt = "{0}{1} er {2}."
+            opening_hours = opening_hours.replace("opið", "opinn")
+            index = opening_hours.find("daga") + 4
+            # Find opening hours from the string
+            times = opening_hours[index:].replace(" ", "")
+            open_time = times[: times.find("-")]
+            close_time = times[times.find("-") + 1 :]
+            voice_opening_hours = ""
+            if index != -1:
+                voice_opening_hours = opening_hours[:index] + " frá klukkan "
+                opening_hours = (
+                    opening_hours[:index]
+                    + " frá klukkan "
+                    + opening_hours[index:].replace(" ", "")
+                )
+            voice_opening_hours = (
+                voice_opening_hours
+                + number_to_text(open_time, case="þf")
+                + " til "
+                + number_to_text(close_time, case="þf")
+            )
+            answer = answ_fmt.format(
+                ans_start,
+                street_name,
+                opening_hours,
+            )
+            voice = answ_fmt.format(
+                ans_start,
+                voice_street_name,
+                voice_opening_hours,
+            )
+        else:
+            return gen_answer("Ekki tókst að sækja opnunartíma fyrir hraðbankann.")
+    elif result.qkey == "AtmFurtherInfoForeignExchange":
+        # Check if atm accepts foreign exchange
+        if atm_list[0]["services"]["foreign_exchange"]["active"] is True:
+            currencies_str: str = _get_foreign_exchange_string(atm_list)
+            ans_start = "Hægt er að kaupa "
+            answ_fmt: str = "{0}{1} í hraðbankanum við {2}."
+            answer: str = answ_fmt.format(
+                ans_start,
+                currencies_str,
+                street_name,
+            )
+            voice: str = answ_fmt.format(
+                ans_start,
+                currencies_str,
+                voice_street_name,
+            )
+        else:
+            answ_fmt: str = (
+                "Ekki er hægt að kaupa erlendan gjaldeyri í hraðbankanum við {0}."
+            )
+            answer: str = answ_fmt.format(
+                street_name,
+            )
+            voice: str = answ_fmt.format(
+                voice_street_name,
+            )
+    elif result.qkey == "AtmFurtherInfoCoinmachine":
```

Comment:
_Very minor point:_ It may be a good idea to make the qkeys referenced in this function constants, and reference those (also in the corresponding nonterminal functions). Just decreases the likelihood of minor typos suddenly eradicating functionality from a query module (those can be a joy to debug :smiling_face_with_tear:).

Reply:
Fixed. Added constants for the qkeys.